encrypt-config/tests/CMakeLists.txt

```diff
@@ -32,7 +32,7 @@ foreach(testfile ${ENCRYPT_CONFIG_TESTS})
   target_include_directories(${testfilename} BEFORE PRIVATE "${CMAKE_SOURCE_DIR}/libminifi/test")
 
   target_wholearchive_library(${testfilename} minifi)
-  target_link_libraries(${testfilename} ${CATCH_MAIN_LIB})
+  target_link_libraries(${testfilename} ${CATCH_MAIN_LIB} ${TEST_BASE_LIB})
```

Comment:
Is this a new library, or why is this addition necessary?

Reply:
It's needed because `encrypt-config/tests/ConfigFileTests.cpp` now uses `TestController`, which is in `test_base` (= `${TEST_BASE_LIB}`.  Before this change, `ConfigFileTests` created the temporary directory directly using `utils::file::create_temp_directory()` -- which also means that this test directory was not cleaned up after the test.

Reply:
I see, thanks.